src/topology/metric_space/gluing.lean

```diff
@@ -162,12 +164,12 @@ private lemma glue_dist_triangle (Φ : γ → α) (Ψ : γ → β) (ε : ℝ)
   end
 | (inl x) (inr y) (inl z) := real.le_of_forall_epsilon_le $ λδ δpos, begin
     have : ∃a ∈ range (λp, dist x (Φ p) + dist y (Ψ p)), a < infi (λp, dist x (Φ p) + dist y (Ψ p)) + δ/2 :=
-      exists_lt_of_cInf_lt (by simp [_inst_1]) (by rw [infi]; linarith),
+      exists_lt_of_cInf_lt (by simp [_inst_3]) (by rw [infi]; linarith),
```

Comment:
Not super important, but maybe we should give this an explicit name?

Reply:
I have used French quotes instead.